### PR TITLE
[ Staking ] add APR calculation based on validators comissions

### PR DESCRIPTION
**Pull request by dpereskokov**
Fixes: #676

### CODE REVIEW

File: services/assets/client.go [hunk 1]
```diff
@@ -64,6 +64,8 @@ func NormalizeValidators(validators []blockatlas.Validator, assets []AssetValida
 }
 
 func NormalizeValidator(plainValidator blockatlas.Validator, validator AssetValidator, coin coin.Coin) blockatlas.StakeValidator {
+	stakingDetails := plainValidator.Details
+	stakingDetails.Reward.Annual = (stakingDetails.Reward.Annual * (100 - validator.Payout.Commission)) / 100
```
Comment (vikmeup) [inline services/assets/client.go]: can we move this logic into separate function? and add test

**Comment by vikmeup** [comment]
Can you also update tezos client for this logic?

**Comment by dpereskokov** [comment]
It works with Tezos too
image

File: services/assets/client.go [hunk 2]
```diff
@@ -64,6 +64,8 @@ func NormalizeValidators(validators []blockatlas.Validator, assets []AssetValida
 }
 
 func NormalizeValidator(plainValidator blockatlas.Validator, validator AssetValidator, coin coin.Coin) blockatlas.StakeValidator {
+	stakingDetails := plainValidator.Details
+	stakingDetails.Reward.Annual = calcAnnual(stakingDetails.Reward.Annual, validator.Payout.Commission)
```
Comment (vikmeup) [inline services/assets/client.go]: maybe just `details` ?

File: services/assets/client.go [hunk 3]
```diff
@@ -73,10 +75,14 @@ func NormalizeValidator(plainValidator blockatlas.Validator, validator AssetVali
 			Image:       GetImage(coin, plainValidator.ID),
 			Website:     validator.Website,
 		},
-		Details: plainValidator.Details,
+		Details: stakingDetails,
 	}
 }
 
+func calcAnnual(coinAnnual float64, commission float64) float64 {
+	return (coinAnnual * (100 - commission)) / 100
```
Comment (vikmeup) [inline services/assets/client.go]: `coinAnnual ` => `annual` ?

**Pull request merged**